odata_query/sqlalchemy/shorthand.py

```diff
@@ -1,10 +1,10 @@
 from typing import List
 
```

Comment:
There are some linting issues here. We use `isort` to keep imports organized and `black` to automatically format code. Most of this is automated, and described here: https://github.com/gorilla-co/odata-query/blob/master/CONTRIBUTING.rst#local-development

Reply:
I noticed that the Github Workflow wasn't running on Pull Requests, so sorry if this was unclear.
I configured the Github Workflow to run on future PRs to get some automated feedback.

Reply:
My apologies again I did use isort but I think I had different ``isort.cfg`` so I didn't endup with the same sort pattern as yours.